### PR TITLE
Added component to display top artists by year

### PR DESCRIPTION
**Pull request by rterker**
When user selects year, the component renders (conditionally) showing top artists for that year. Right now, the component appears instantly. May want to add an effect or transition to this later. This is a starting point, not the finished product.

### CODE REVIEW

File: client/src/components/TopAristsByYearComp.jsx
Comment (keithgibson) [inline client/src/components/TopAristsByYearComp.jsx:1]: Guessing the 'album' appearances here are meant to be 'artist'?

File: client/styles/index.scss
Comment (keithgibson) [inline client/styles/index.scss:1]: ditto here. unless I'm missing something

File: client/src/components/TopAristsByYearComp.jsx
Comment (rterker) [reply client/src/components/TopAristsByYearComp.jsx:1]: Yes artists

File: client/styles/index.scss
Comment (rterker) [reply client/styles/index.scss:1]: Artists

**Review by NinaSkyttis — APPROVED** [review]

**Pull request merged**